docs/source/routing/performance/caching/index.mdx

```diff
@@ -1,16 +1,36 @@
 ---
-title: Caching
-subtitle: Accelerate query retrieval with GraphOS caching.
+title: Caching in GraphOS Router
```

Comment:
Considering the free-and-open router binary also has some (in-memory) caching, this may be more accurate:

```suggestion
title: Caching in Apollo Router
```